shell/platform/tizen/tizen_surface_gl.h

```diff
@@ -39,6 +39,7 @@ class TizenSurfaceGL : public TizenSurface {
   bool display_valid_ = false;
   EGLDisplay egl_display_ = EGL_NO_DISPLAY;
   EGLSurface egl_surface_ = nullptr;
+  EGLSurface egl_resource_surface_ = nullptr;
```

Comment:
For consistency, how about changing nullptr to EGL_NO_SURFACE?

Reply:
done